tools/marvin/marvin/config/test_data.py

```diff
@@ -781,6 +781,15 @@
         "ostype": "CentOS 5.6 (64-bit)"
 
     },
+    "coreos_volume": {
+        "diskname": "Volume_core",
+        "urlvmware":"http://10.147.28.7/templates/coreos/coreos_production_vmware.ova",
```

Comment:
If you add download links like this, no one else in the community will be able to run your test. Please check if you can use something that is available already, check openvm.eu for example.